ietf-microwave-radio-link.yang

```diff
@@ -121,6 +126,45 @@ module ietf-microwave-radio-link {
        for configuration or status.";
   }
 
+  grouping header-compression-profile {
```

Comment:
This grouping is used only once. YANG doctors usually complain and ask the authors to consider whether there is a need to make this grouping re-usable by other modules or, otherwise, move the data nodes within the code

Reply:
Having the group the module is likely more readable than having data nodes within the code, anyway no problem to move data nodes into the code.